packages/website/src/demo-building-blocs/ExampleLayout.tsx

```diff
@@ -15,16 +16,22 @@ export interface ExampleLayoutProps {
     section: string;
     code: string;
     examples: Record<string, {code: string; title: string}>;
+    /**
```

Comment:
very nice addition, I know I would have put the wrong path at some point :see_no_evil: